app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -191,10 +191,8 @@ def process_invalid_webauthn(form)
         else
           flash.now[:error] = t('errors.webauthn_setup.unique_name')
         end
-      elsif form.platform_authenticator?
-        flash[:error] = t('errors.webauthn_platform_setup.general_error')
       else
-        flash[:error] = t(
+        flash.now[:error] = t(
```

Comment:
Why are we changing this from `flash` to `flash.now` ?

Reply:
I got an error that specifically said it should be flash.now. I'm not sure the why though.